tools/codegen/codespec/merge_attributes.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 // mergeNestedAttributes recursively merges nested attributes
-func mergeNestedAttributes(existingAttrs *Attributes, newAttrs Attributes, computability ComputedOptionalRequired, reqBodyUsage AttributeReqBodyUsage, isFromResponse bool) {
```

Comment:
relying on parent property computability was causing errors once processing patch request properties, was able to remove this property